guide/english/miscellaneous/create-and-upload-and-link-animated-gif-image/index.md

```diff
@@ -7,4 +7,4 @@ title: Create and Upload and Link Animated Gif Image
     *   Crop, Resize and Optimize as many times or as required.
 4.  Download animated GIF.
 5.  Upload image to <a href='http://imgur.com' target='_blank' rel='nofollow'>imgur.com</a>.
-6.  Link image using `![image title](https://i.imgur.com/RqgGcf5.gif)`.
\ No newline at end of file
+6.  Link image using `![image title](https://cdn-media-1.freecodecamp.org/imgr/RqgGcf5.gif)`.
```

Comment:
Same over here. I guess it is best to leave this as an imgur link, since no user would ever be uploading to fCC's CDN

Reply:
@thecodingaviator Thank you for the catch.

Reply:
We should ultimately encourage users to upload photos to another website entirely.

Reply:
@ahmadabdolsaheb I think that's what should be done. Please open an issue. Rest looks good now